packages/sdk/src/modules/FundOperations.ts

```diff
@@ -80,17 +80,9 @@ export function withFundOperations<TBase extends MidasBaseConstructor>(Base: TBa
       return await this.mint(cTokenAddress, amount);
     }
 
-    async repay(cTokenAddress: string, underlyingTokenAddress: string, isRepayingMax: boolean, amount: BigNumber) {
+    async repayBorrow(cTokenAddress: string, isRepayingMax: boolean, amount: BigNumber) {
```

Comment:
Why do we now have two repay functions?
`repayBorrow` & `repay`?

Aren't we always repaying a borrowed token?

Reply:
`repay` = `approved` + `repayBorrow`. So `repay` has 2 steps for transactions, it needs to be separated to show tx to users transparently